src/tools/genpolicy/genpolicy-settings.json

```diff
@@ -136,6 +136,17 @@
     },
     "volumes": {
         "emptyDir": {
+            "mount_type": "local",
+            "mount_source": "^$(cpath)/$(sandbox-id)/rootfs/local/",
```

Comment:
These paths might need to change when using confidential=true at MSFT. They are probably specific to main code on Mariner, *without* confidential enabled.